src/unix/ibmi.c

```diff
@@ -166,10 +234,11 @@ uint64_t uv_get_free_memory(void) {
   uint64_t current_unprotected_storage_used =
     rcvr.current_unprotected_storage_used * 1024ULL;
 
-  uint64_t free_storage_size =
-    (main_storage_size - current_unprotected_storage_used) * 1024ULL;
+  /* It is possible on IBMi due to 'single-level storage' */
```

Comment:
What is possible?

Reply:
> What is possible?

[Current unprotected storage used](https://www.ibm.com/support/knowledgecenter/en/ssw_ibm_i_72/apis/qwcrssts.htm) is the total amount of storage in use for temporary objects, including memory and disks. As far as I know, there is no such an API to get the **memory-only** usage of the system.

Reply:
Okay, could you maybe add to the comment something like "Current unprotected storage includes the storage used for memory and disks so it is possible to exceed the amount of main storage"? Without any sort of context in the current comment it's not clear what "It" refers to.

Reply:
Sure